### PR TITLE
Add OwO version of English

### PR DESCRIPTION
**Pull request by Lily-129**
:eggTF:

### CODE REVIEW

File: eod/translation/languages/en_owo.json [hunk 1]
```diff
@@ -0,0 +1,209 @@
+{
+  "Name": "Engwish (US)",
```
Comment (Nv7-GitHub) [inline eod/translation/languages/en_owo.json:2]: Can you make it `Engwish (Owo)`?

**Review by Nv7-GitHub — CHANGES_REQUESTED** [review]

File: eod/translation/languages/en_owo.json (hunk 1 above)
Comment (Lily-129) [reply eod/translation/languages/en_owo.json]: I made the change. Do I have to recreate the pull request for it to apply?

Comment (rayzchen) [reply eod/translation/languages/en_owo.json]: no

**Review by Nv7-GitHub — APPROVED** [review]
Seems good!

**Pull request merged**